docs/data/osdf/install-cache-rpm.md

```diff
@@ -0,0 +1,125 @@
+title: Installing the OSDF Cache by RPM
+
+Installing the OSDF Cache
+=========================
+
+This document describes how to install a Pelican-based Open Science Data Federation (OSDF) Cache service via RPMs.
+This service allows a site or regional network to cache data 
+
+!!! note
+    The cache must be registered with the OSG prior to joining the data federation.
+    You may start the registration process prior to finishing the installation by [using this link](#registering-the-cache) 
+    along with information like:
+
+    * Resource name and hostname
+    * Administrative and security contact(s)
+
+
+Before Starting
+---------------
+
+Before starting the installation process, consider the following requirements:
+
+* __Operating system:__ A RHEL 8 or RHEL 9 or compatible operating systems.
+* __User IDs:__ If they do not exist already, the installation will create the Linux user named `xrootd` for running daemons.
+* __File Systems:__ The cache should have a partition of its own for storing data and metadata.
+* __Host certificate:__ Required for authentication.  See note below.
+* __Network ports:__ The cache service requires the following ports open:
+  * Inbound TCP port 8443 for file access via the HTTP(S) and XRoot protocols.
+  * (Optional) Inbound TCP port cache for access to the web interface for monitoring and configuration;
+    if enabled, this should be restricted to the LAN or management network.
+* __Hardware requirements:__ We recommend that an origin has at least 1Gbps connectivity and 12GB of RAM.
```

Comment:
Should we say something about NVMe?

Reply:
These requirements need to be updated -- what's listed is way too small these days.

Reply:
Can you look at #200? That's the PR I had made for updating the cache requirements.